content/reference/reader.adoc

```diff
@@ -209,6 +209,35 @@ by invoking the Var `#'my.project.foo/bar` on the vector `[1 2 3]`. The data rea
 
 Reader tags without namespace qualifiers are reserved for Clojure. Default reader tags are defined in https://clojure.github.io/clojure/clojure.core-api.html#clojure.core/default-data-readers[default-data-readers] but may be overridden in `data_readers.clj` or by rebinding https://clojure.github.io/clojure/clojure.core-api.html#clojure.core/%2Adata-readers%2A[pass:[*data-readers*]]. If no data reader is found for a tag, the function bound in https://clojure.github.io/clojure/clojure.core-api.html#clojure.core/%2Adefault-data-reader-fn%2A[pass:[*default-data-reader-fn*]] will be invoked with the tag and value to produce a value. If pass:[*default-data-reader-fn*] is nil (the default), a RuntimeException will be thrown.
 
+Clojure 1.4 introduced some tagged literals useful for Java interop. So far, these include instants and UUIDs.
+
+Instants have the format `#inst "yyyy-mm-ddThh:mm:ss.fff+hh:mm"`. The default reader will parse the supplied string into a
```

Comment:
Note that while this is the full format, most of the parts are optional. So `#inst "2018"` is a valid inst literal as is `#inst "2018-01-01"` etc. Might need to check the code for allowed options.

Reply:
Are you asking *me* to check the code, or that I should add that note to the docs?